src/components/Account/MainLayout.js

```diff
@@ -29,26 +29,14 @@ const useStyles = createStyles((theme) => ({
 
 const AccountMainLayout = ({ pageName, children, pageTitle, description }) => {
   const { user } = useMoralis();
-  const [email, setEmail] = useState(null);
   const { classes } = useStyles();
-  const getEmail = useCallback(() => {
-    if (user?.attributes?.email) {
-      setEmail(user.attributes.email);
-    }
-  }, [user]);
-
-  useEffect(() => {
-    if (!email) {
-      getEmail();
-    }
-  }, [email, getEmail]);
   return (
     <Layout withAuth pageTitle={pageTitle} description={description}>
       <Flex className={classes.container} direction='row'>
         <Flex className={classes.accountOverviewBoxContainer}>
           <AccountOverviewBox
             pageName={pageName ?? pageTitle}
-            email={email}
```

Comment:
Also removed this bit of code since we don't need to put user's email in a state. We can use it directly from Moralis' `user` obj. Other than removing this code, another advantage is that the email will be updated immediately in the view once the email gets by the user. 

Previously, a refresh has to be made to see the change in the email because our now removed `email` state isn't updated.